### DISCUSSION TITLE
How can I grant permission for other than photos/videos within DCIM?

### DISCUSSION THREAD

**Discussion by MerkuR-92** [anilbeesetti/nextplayer#1021] [Q&A]
When installing the app (whether from the Play Store or directly the APK from here), it only allows me to grant access permission for photos and videos, but not for the rest of the system folders.

I'm using a Galaxy S24 and in special permissions where "access to all files" is listed, Next Player does not appear (but others like Material Files do).

Is there something I'm missing?

**Comment by anilbeesetti** [comment]
Access to all files is a high-level permission. For that permission to work, we need approval from the Play Store. Since this is a video player, access to videos would be sufficient for the app to work, and there's no need for all files permission. By the way, why do you need to enable all files permission?

**Reply by MerkuR-92** [reply]
Because I need to access videos from specific folders that I have in other parts of the system (this way, I would have the folder in question as a shortcut every time I enter Next Player), it's a feature I had with the other app I used before but it stopped receiving support on Android (nPlayer), just like the ability to lock specific folders with biometrics.

Thanks for the response.
